lambda/exportPass/index.js

```diff
@@ -20,8 +20,9 @@ exports.handler = async (event, context) => {
     }
     if (event.queryStringParameters.facilityName && event.queryStringParameters.park) {
 
-      const tokenObj = await checkPermissions(event);
```

Comment:
`tokenObj` is referenced later in this script (line 104, eg) but is now left undefined.

Reply:
Fixed in bea2fd8